src/Autosuggest.js

```diff
@@ -235,15 +252,19 @@ class Autosuggest extends Component {
           case 'Enter': {
             const focusedSuggestion = this.getFocusedSuggestion();
 
-            if (focusedSuggestion !== null) {
+            if (focusedSuggestion) {
```

Comment:
@rickhanlonii Yeah, I see :)

What I don't feel comfortable about is that `UPDATE_FOCUSED_SUGGESTION` is triggered too much, even when we don't update the focused suggestion.

What do you think about calling `maybeSelectFirstSuggestion()` in `componentWillReceiveProps()` instead?
Have a look [here](https://github.com/moroshko/react-autosuggest/commit/c2cc05484a0b8a459f65f115177aa8be5dec7d90).

It is more code, but I think it's cleaner.
What do you think?